offchain-modules/src/packages/db/entity/TronLock.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  VersionColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+  Column,
+  Index,
+  PrimaryColumn,
+} from 'typeorm';
+
+export type TronAssetType = 'trx' | 'trc10' | 'trc20';
+
+@Entity()
+@Index(['tronLockTxHash', 'tronLockIndex'], { unique: true })
+export class TronLock {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Index()
```

Comment:
This index is redundant if there is unique index for `['tronLockTxHash', 'tronLockIndex']`